src/api/flaskr/service/shifu/route.py

```
@@ -30,7 +30,7 @@
 from flaskr.route.common import make_common_response
 from flaskr.framework.plugin.inject import inject
 from flaskr.service.common.models import raise_param_error, raise_error
-from .const import UNIT_TYPE_TRIAL
+from ..lesson.models import LESSON_TYPE_TRIAL
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)